### PR TITLE
Webhooks create page

### PR DESCRIPTION
**Pull request by sans-harness**
- Adds the `create-webhooks` page to playground. 

https://github.com/user-attachments/assets/dee8ce3f-3806-48ce-b81a-ad74072b7d98

### CODE REVIEW

File: packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    <Input id="payloadUrl" {...register!('payloadUrl')} placeholder="https://example.com/harness" />
+    {errors!.payloadUrl && (
+      <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors!.payloadUrl.message?.toString()}</FormFieldSet.Message>
+    )}
+  </FormFieldSet.ControlGroup>
+)
+
+export const WebhookSecretField: React.FC<FieldProps> = ({ register, errors }) => (
+  <FormFieldSet.ControlGroup>
+    <FormFieldSet.Label htmlFor="secret" required>
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx:69]: should secret be required? github allows webhooks without a secret. does gitness not allow the same?

File: packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            />
+          </div>
+        }
+        right
+      />
+    </StackedList.Item>
+  </StackedList.Root>
+)
+
+export const WebhookNameField: React.FC<FieldProps & { disabled: boolean }> = ({ register, errors, disabled }) => (
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx:31]: why do we need so much boilerplate for all the form fields? are we not using the schema framework which can render basic type inputs out-of-the-box?
cc @srdjan-harness

File: packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+export const WebhookSSLVerificationField: React.FC<FieldProps> = ({ watch, setValue }) => {
+  const sslVerificationValue = watch!('sslVerification')
+  const handleAccessChange = (value: string) => {
+    setValue!('sslVerification', value as '1' | '2')
+  }
+
+  return (
+    <FormFieldSet.Root className="mb-0">
+      <FormFieldSet.ControlGroup>
+        <FormFieldSet.Label htmlFor="sslVerification" required>
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/components/webhooks/create-webhooks-form-fields.tsx:88]: same here. why is this required?

File: packages/playground/src/components/webhooks/create-webhook-form-data.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  { id: 3, event: 'Branch deleted' }
+]
+
+export const tagEvents: WebhookEvent[] = [
+  { id: 4, event: 'Tag created' },
+  { id: 5, event: 'Tag updated' },
+  { id: 6, event: 'Tag deleted' }
+]
+
+export const prEvents: WebhookEvent[] = [
```
Comment (abhinavrastogi-harness) [inline packages/playground/src/components/webhooks/create-webhook-form-data.ts:15]: can we reuse the enums from the types file instead of duplicating all the strings?

Comment (sans-harness) [reply packages/playground/src/components/webhooks/create-webhook-form-data.ts:15]: Already fixed in followup PR
